modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,80 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+* The current release and update target releases have already been mirrored to a locally accessible registry.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} CLI:
+
+. Configure the OpenShift Update Service target namespace, for example, `openshift-update-service`:
++
+[source,terminal]
+----
+$ NAMESPACE=openshift-update-service
+----
++
+The namespace must match the `targetNamespaces` value from the OperatorGroup.
+
+. Configure the name of the OpenShift Update Service application, for example, `service`:
++
+[source,terminal]
+----
+$ NAME=service
+----
+
+. Configure the local registry and repository for the release images, for example, `registry.example.com/openshift/release`:
```

Comment:
Need the similar change [1] as you did in the **Creating an OpenShift Update Service application by using the web console**

[1] In the Releases field, enter the local registry and repository created to contain the release images in Mirroring the OpenShift Container Platform image repository, for example, registry.example.com/ocp4/openshift4-release-images.

Reply:
fixed